app/editor/src/features/admin/ingests/configurations/AudioClip.tsx

```diff
@@ -30,7 +31,12 @@ export const AudioClip: React.FC = (props) => {
         options={TimeZones}
         value={timeZone}
       />
-      <FormikText label="Format" name="configuration.format" tooltip="Format of the clip" />
+      <FormikText
```

Comment:
This will allow us to capture multiple A/V feeds in the same source folder.  Before this change a clip would use the first file it found that matched the time period it needed.  Now it can filter by the filename.